encrypted-media/scripts/playback-temporary-multikey-sequential.js

```diff
@@ -37,13 +37,21 @@ function runTest(config,qualifier) {
         function onMessage(event) {
             config.messagehandler(event.messageType, event.message, {variantId: event.target.variantId}).then(function(response) {
                 return event.target.update(response);
+            }).then(function(){
+                if (!_waitingForKey) {
+                    _video.src = URL.createObjectURL(_mediaSource);
```

Comment:
It seems we could get here multiple times since _waitingForKey gets reset below. Even if that's not expected, it seems fragile. You could just check `!video.src` to avoid this.